test/release/compatibility_test.go

```diff
@@ -0,0 +1,23 @@
+package release_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/kumahq/kuma/pkg/version"
+)
+
+var _ = Describe("Compatibility", func() {
+
+	It("current version is defined in CompatibilityMatrix", func() {
+		// given
+		currentVersion := version.Build.Version
+
+		// when
+		dpComptibility, err := version.CompatibilityMatrix.DP(currentVersion)
+
+		// then
+		Expect(err).ToNot(HaveOccurred())
+		Expect(dpComptibility).ToNot(BeNil())
```

Comment:
Might want to add a string of explanation so that folks know they need to update the compat matrix when this test fails.